server/server/server.go

```diff
@@ -57,7 +57,6 @@ func NewServer(opts ...server_options.ServerOption) *Server {
 	authMiddleware := server_options.WithAPIMiddleware(([]api.Middleware{
 		headers.WithForwardHeaders(
 			[]string{
-				echo.HeaderAuthorization,
```

Comment:
let's add a comment here clarifying that the authorization header is forwarded by grpc-gateway. Ideally we would  use `runtime.NewServeMux(runtime.WithOutgoingHeaderMatcher..` though this is not a priority now since the forwarding already works